### ISSUE TITLE
Enum not working in schema validation

### ISSUE THREAD

**Issue by lukas-brenning** [codedawi/pytest-schema#1]
Thanks a lot for this cool repo, I have an issue whenever I want to use Enum, I'm running into a TypeError. I used eventually the example from the docs and even there it doesn't work as expected. 

```>>> def test_users_endpoint():
...     response = [
...         {
...             "id": 2,
...             "name": "Sue",
...             "age": 28,
...             "email": "sue@gmail.com",
...             "gender": "female",
...             "role": "admin",
...             "friends": [5, 6],
...             "address": {
...                 "street": "123 Washington Ave.",
...                 "city": "New York",
...                 "state": "NY",
...                 "zipcode": "099012",
...             }
...         },
...         {
...             "id": 5,
...             "name": "Sam",
...             "age": 42,
...             "email": "sam@aol.com",
...             "role": "user",
...             "friends": [2, 6, 7],
...             "address": {
...                 "street": "5 Sunset St.",
...                 "street2": "Apt # 55-b",
...                 "city": "San Jose",
...                 "state": "CA",
...                 "zipcode": "054053",
...             }
...         }]
...     assert schema(users) == response
...
>>> test_users_endpoint()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<stdin>", line 33, in test_users_endpoint
  File "/opt/apps/k2view/.local/lib/python3.6/site-packages/pytest_schema/schema.py", line 45, in __eq__
    return bool(self.validate(value))
  File "/opt/apps/k2view/.local/lib/python3.6/site-packages/schema.py", line 366, in validate
    return type(data)(o.validate(d, **kwargs) for d in data)
  File "/opt/apps/k2view/.local/lib/python3.6/site-packages/schema.py", line 366, in <genexpr>
    return type(data)(o.validate(d, **kwargs) for d in data)
  File "/opt/apps/k2view/.local/lib/python3.6/site-packages/schema.py", line 169, in validate
    [self._error.format(data) if self._error else None] + errors,
schema.SchemaError: Or({'id': <class 'int'>, 'name': <class 'str'>, 'description': Or(None, <class 'str'>), 'email': Regex('.*?@.*?\\.[A-Za-z]{2,6}'), 'age': And(<class 'int'>, <function <lambda> at 0x7fe8e66ef598>), Optional('gender'): <class 'str'>, 'role': Enum(['user', 'super-user', 'admin']), 'friends': [<class 'int'>], 'address': {'street': <class 'str'>, Optional('street2'): <class 'str'>, 'city': <class 'str'>, 'state': And(<class 'str'>, <function <lambda> at 0x7fe8e66ef620>), 'zipcode': <class 'str'>}}) did not validate {'id': 2, 'name': 'Sue', 'age': 28, 'email': 'sue@gmail.com', 'gender': 'female', 'role': 'admin', 'friends': [5, 6], 'address': {'street': '123 Washington Ave.', 'city': 'New York', 'state': 'NY', 'zipcode': '099012'}}
Key 'role' error:
Enum(['user', 'super-user', 'admin']).validate('admin') raised TypeError("__init__() got an unexpected keyword argument 'error'",)```

**Comment by codedawi** [comment]
My apologies just seeing this now. I will look into it!

**Comment by codedawi** [comment]
Released in version [`v0.1.1`](https://github.com/codedawi/pytest-schema/releases/tag/v0.1.1)
